src/encoded/commands/generate_ontology.py

```diff
@@ -5,6 +5,7 @@
 import re
```

Comment:
Changes (to use new IngestionConnection) in this file are no longer strictly required, but for future.